rootconf/default/etc/init.d/dropbear.sh

```diff
@@ -14,31 +15,18 @@ daemon=/usr/sbin/$name
 
 [ -x $daemon ] || exit 0
 
-ARGS="-m -B"
+ARGS="-m -B -R"
```

Comment:
I don't think we need the -R, as our `get_keys()` implementation makes sure the keys exist.